zerver/openapi/zulip.yaml

```diff
@@ -12999,6 +12999,21 @@ paths:
                     **Changes**: New in Zulip 8.0 (feature level 235).
                   type: boolean
                   example: true
+                resolved_topic_notice_auto_read_policy:
+                  description: |
+                    Controls whether the resolved-topic notices are marked as read.
+
+                    - "always" - Always mark resolved-topic notices as read.
+                    - "except_followed" - Mark resolved-topic notices as read in topics not followed by the user.
+                    - "never" - Never mark resolved-topic notices as read.
+
+                    **Changes**: New in Zulip 11.0 (feature level ZF-e0d354.md).
```

Comment:
You need it to say `New in Zulip 11.0 (feature level ZF-e0d354)`, without the `md`, for `tools/merge-api-changelogs` to work :). I fixed that before running it.